pipelines/ocw/pipeline-theme-assets.yml

```diff
@@ -0,0 +1,65 @@
+---
+resource_types:
+  - name: rclone
+    type: docker-image
+    source:
+      repository: mitodl/concourse-rclone-resource
+      tag: latest
+resources:
+  - name: ol-infrastructure
+    type: git
+    source:
+      uri: https://github.com/mitodl/ol-infrastructure.git
+      branch: ((ol-infrastructure-git-ref))
+  - name: ocw-hugo-themes
+    type: git
+    source:
+      uri: https://github.com/mitodl/ocw-hugo-themes.git
+      branch: ((ocw-hugo-theme-git-ref))
+  - name: ocw-artifacts
+    type: rclone
+    source:
+      config: |
+        [s3-remote]
+        type = s3
+        provider = AWS
+        env_auth = true
+        region = us-east-1
+
+jobs:
+  - name: build-theme-assets
+    serial: true
+    plan:
+      - get: ol-infrastructure
```

Comment:
There's actually no need to have the ol-infrastructure repo involved in this pipeline at all so you can remove this and the resource definition for it, as well as the inputs line below.

Reply:
👍